commit fa5ef4f6f35fd64ecbb447a7e3380641574c4b4e
Author: Giulio Eulisse <giulio.eulisse@cern.ch>
Date:   Thu Apr 22 11:58:10 2010 +0200

    Use rpmhash_big.c and rpmhash_big.h in place of rpmhash.C and rpmhash.H

diff --git a/lib/depends.c b/lib/depends.c
index ef55adb..c41e0af 100644
--- a/lib/depends.c
+++ b/lib/depends.c
@@ -33,8 +33,8 @@ static rpmds rpmlibP = NULL;
 #define HASHTYPE depCache
 #define HTKEYTYPE const char *
 #define HTDATATYPE int
-#include "lib/rpmhash.H"
-#include "lib/rpmhash.C"
+#include "lib/rpmhash_big.h"
+#include "lib/rpmhash_big.c"
 
 /**
  * Compare removed package instances (qsort/bsearch).
diff --git a/lib/fprint.c b/lib/fprint.c
index c04c8af..a94de82 100644
--- a/lib/fprint.c
+++ b/lib/fprint.c
@@ -14,7 +14,7 @@
 #include <libgen.h>
 
 /* Create new hash table type rpmFpEntryHash */
-#include "lib/rpmhash.C"
+#include "lib/rpmhash_big.c"
 
 #undef HASHTYPE
 #undef HTKEYTYPE
@@ -22,7 +22,7 @@
 #define HASHTYPE rpmFpEntryHash
 #define HTKEYTYPE const char *
 #define HTDATATYPE const struct fprintCacheEntry_s *
-#include "lib/rpmhash.C"
+#include "lib/rpmhash_big.c"
 
 fingerPrintCache fpCacheCreate(int sizeHint)
 {
diff --git a/lib/fprint.h b/lib/fprint.h
index 4ac7dad..6c16fee 100644
--- a/lib/fprint.h
+++ b/lib/fprint.h
@@ -35,7 +35,7 @@ const char * baseName;	/*!< file base name */
 #define HASHTYPE rpmFpEntryHash
 #define HTKEYTYPE const char *
 #define HTDATATYPE const struct fprintCacheEntry_s *
-#include "lib/rpmhash.H"
+#include "lib/rpmhash_big.h"
 
 /**
  * Finger print cache entry.
@@ -70,7 +70,7 @@ struct rpmffi_s {
 #define HASHTYPE rpmFpHash
 #define HTKEYTYPE const fingerPrint *
 #define HTDATATYPE struct rpmffi_s
-#include "lib/rpmhash.H"
+#include "lib/rpmhash_big.h"
 
 /** */
 #define	FP_ENTRY_EQUAL(a, b) (((a)->dev == (b)->dev) && ((a)->ino == (b)->ino))
diff --git a/lib/rpmal.c b/lib/rpmal.c
index b320ca1..b5dfd33 100644
--- a/lib/rpmal.c
+++ b/lib/rpmal.c
@@ -62,8 +62,8 @@ struct fileIndexEntry_s {
 #define HASHTYPE rpmalProvidesHash
 #define HTKEYTYPE const char *
 #define HTDATATYPE struct availableIndexEntry_s
-#include "lib/rpmhash.H"
-#include "lib/rpmhash.C"
+#include "lib/rpmhash_big.h"
+#include "lib/rpmhash_big.c"
 
 #undef HASHTYPE
 #undef HTKEYTYPE
@@ -71,8 +71,8 @@ struct fileIndexEntry_s {
 #define HASHTYPE rpmalFileHash
 #define HTKEYTYPE struct fileNameEntry_s
 #define HTDATATYPE struct fileIndexEntry_s
-#include "lib/rpmhash.H"
-#include "lib/rpmhash.C"
+#include "lib/rpmhash_big.h"
+#include "lib/rpmhash_big.c"
 
 
 
diff --git a/lib/transaction.c b/lib/transaction.c
index e4f282b..e5e863f 100644
--- a/lib/transaction.c
+++ b/lib/transaction.c
@@ -830,8 +830,8 @@ static void skipInstallFiles(const rpmts ts, rpmte p)
 
 #define HASHTYPE rpmStringSet
 #define HTKEYTYPE const char *
-#include "lib/rpmhash.H"
-#include "lib/rpmhash.C"
+#include "lib/rpmhash_big.h"
+#include "lib/rpmhash_big.c"
 
 /* Get a rpmdbMatchIterator containing all files in
  * the rpmdb that share the basename with one from
